.ci/Jenkinsfile

```
@@ -124,6 +124,9 @@ def generateStep(Map params = [:]){
       try {
         deleteDir()
         unstash 'build'
+        dir("${BASE_DIR}") {
+          sh script: """.ci/scripts/install-go.sh "${GO_VERSION}" """, label: 'Install Go'
```

`${var}` expressions are evaluated before to launch the command, so the following step if `GO_VERSION=1.12.3`

```suggestion
          sh script: ".ci/scripts/install-go.sh '${GO_VERSION}' ", label: 'Install Go'
```

it will run `.ci/scripts/install-go.sh '1.12.3' `, becasue the expresion was evaluated on Groovy time.